### ISSUE TITLE
PSA tests should use a temporary directory

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#557]
@sseyler can you please modify the tests for PSA so that they do not write the `./psadata` directory to the current directory but instead use a temporary directory, possibly using `tempdir`? I think you just have to add the [targetdir](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/analysis/psa.py#L1078) keyword somewhere.

When I manually run
```
./mda_nosetests -v --processes=1 --process-timeout=120 ./analysis/test_psa.py

Ran 3 tests in 7.313s

FAILED (errors=3)
```

I currently get errors like the following
```
ERROR: test_reversal_hausdorff (MDAnalysisTests.analysis.test_psa.TestPSAnalysis)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nfs/homes2/oliver/.local/lib/python2.7/site-packages/MDAnalysisTests-0.13.0.dev0-py2.7.egg/MDAnalysisTests/analysis/test_psa.py", line 57, in tearDown
    shutil.rmtree('psadata') # remove psa data directory
  File "/usr/lib/python2.7/shutil.py", line 247, in rmtree
    rmtree(fullname, ignore_errors, onerror)
  File "/usr/lib/python2.7/shutil.py", line 256, in rmtree
    onerror(os.rmdir, path, sys.exc_info())
  File "/usr/lib/python2.7/shutil.py", line 254, in rmtree
    os.rmdir(path)
OSError: [Errno 39] Directory not empty: 'psadata/fitted_trajs'
```

I am not sure if these errors will go away, but using temporary directories would be a start.

**Comment by orbeckst** [comment]
See comments in #551 which possibly relate to this issue.

**Comment by orbeckst** [comment]
closed via #575
